relay-event-schema/src/protocol/span_v2.rs

```diff
@@ -0,0 +1,367 @@
+use relay_protocol::{Annotated, Empty, FromValue, IntoValue, Object, Value};
+
+use std::fmt;
+
+use serde::Serialize;
+
+use crate::processor::ProcessValue;
+use crate::protocol::{Attribute, SpanId, Timestamp, TraceId};
+
+use super::OperationType;
+
+/// A version 2 (transactionless) span.
+#[derive(Clone, Debug, Default, PartialEq, Empty, FromValue, IntoValue)]
+pub struct SpanV2 {
+    #[metastructure(required = true, trim = false)]
+    pub trace_id: Annotated<TraceId>,
+
+    /// The ID of the span enclosing this span.
+    pub parent_span_id: Annotated<SpanId>,
+
+    /// The ID of the trace the span belongs to.
+    /// The Span id.
```

Comment:
`id` / `ID` (I think we usually use `id`)?

Also I guess second line is redundant.

Reply:
I think I intended to replace the line, not sure how that happened.